addons/crm/tests/test_crm_lead.py

```diff
@@ -604,6 +604,72 @@ def test_crm_team_alias(self):
         # self.assertFalse(new_team.alias_id.alias_name)
         # self.assertFalse(new_team.alias_name)
 
+    @users('user_sales_manager')
+    def test_crm_team_alias_helper(self):
+        """Test that the help message is the right one if we are on multiple team with different settings."""
+        # archive other teams
+        self.env['crm.team'].search([]).active = False
+
+        other_company = self.env['res.company'].sudo().create({'name': 'OtherCompany'})
```

Comment:
Normally you already have company in mail (check activate multi company), to check in 15.